qutip/tests/core/data/test_norm.py

```diff
@@ -63,9 +63,6 @@ class TestTraceNorm(testing.UnaryOpMixin):
     def op_numpy(self, matrix):
         return np.linalg.svd(matrix)[1].sum()
 
-    # CSR[square,sparse] fails if tol is larger.
-    tol = 3.7e-8
```

Comment:
Does this no longer fail if the tolerance is larger?

Reply:
This one was using only the absolute tolerance instead of both the relative and the absolute one (which is use in `numpy.testing.all_close`).  With the addition of relative tolerance it no longer fails.

Reply:
Nice!